src/test/java/stepDefinition/ShoppingCartPageStepDef.java

```diff
@@ -0,0 +1,69 @@
+package stepDefinition;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.junit.Assert;
+import pages.ShoppingCartPage;
+
+import static helpers.ElementSearchUtils.getElementByName;
+import static helpers.ElementSearchUtils.getPage;
+import static utils.TestUtils.getPageObjectName;
+
+public class ShoppingCartPageStepDef extends AbstractStepDef {
+
+    ShoppingCartPage shoppingCartPage = new ShoppingCartPage(webDriver);
+    String inputQuantity;
+
+    @Then("^'(.*)' product is displayed$")
+    public void productIsDisplayed(String product) {
+        waitForPageLoaded();
+        Assert.assertTrue("The product is not found", getElementByName("productName").isDisplayed());
+    }
+
+    @Then("^'(.*)' notification is displayed$")
+    public void alertMessageIsDisplayed(String message) {
+        Assert.assertTrue("The alert message is not displayed", shoppingCartPage.getAlertMessage().isDisplayed());
+        shoppingCartPage.getCloseAlertMessage().click();
+        waitForPageLoaded();
+    }
+
+    @Then("^'(.*)' product quantity changed correct$")
+    public void checkProductQty(String productName) {
+        Assert.assertEquals("The product quantity doesn't match", inputQuantity, getElementByName("quantityField").getAttribute("value"));
+    }
+
+    @And("^user fills in '(.*)'$")
+    public void fillIn(String itemsQty) {
+        getElementByName("quantityField").clear();
+        waitElementIsClickable(shoppingCartPage.getQuantityField());
+        getElementByName("quantityField").sendKeys(itemsQty);
+        inputQuantity = itemsQty;
+    }
+
+    @When("'(.*)' product quantity changed on (.*)")
+    public void checkProductQuantity(String productName, int qtyValue) {
+        waitVisibility(shoppingCartPage.getQuantityField());
+        String expectedValue = String.valueOf(qtyValue);
+        String actualValue = shoppingCartPage.getQuantityField().getAttribute("value");
+        Assert.assertEquals(expectedValue, actualValue);
+    }
+
+    @And("^user clicks on '(.*)' for the product$")
+    public void clickRemoveCheckBox(String removeCheckBox) {
+        waitElementIsClickable(shoppingCartPage.getRemoveCheckBox());
+        getElementByName(removeCheckBox).click();
+        getPage(getPageObjectName(ShoppingCartPage.class));
+    }
+
+    @When("Total products price is displayed correct")
+    public void checkTotalPrice() {
+        Assert.assertTrue(shoppingCartPage.getProductsPrice().getText().equals(shoppingCartPage.getOrderTotalPrice().getText()));
```

Comment:
assertEquals instead of assertTrue

Reply:
or you can user **assertThat()** and add log message

Reply:
Done.
Thank You!